tycho-gpg-plugin/src/main/java/org/eclipse/tycho/gpg/SignRepositoryArtifactsMojo.java

```diff
@@ -120,6 +120,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                 // skip packed artifacts
                 continue;
             }
+            if(!"osgi.bundle".equals(artifact.getAttribute("classifier"))) {
+                continue; //Don't generate signatures for anything that isn't a "bundles"
+            }
```

Comment:
Instead of a `continue` here, as we may add support for features or other some day, it would be better to use a "positive" condition so we can easily add some `else` clause for other classifiers in the future.

Reply:
I was matching the surrounding code while trying to keep the change as minimal as possible.  Making the change you request would imply also modifying lines 126 & 127 so the path to the jar can change based on the artifact type.

Would it be acceptable to make that change but implement still only generating the path for bundles?

Reply:
yes, I think it would be ok.